### PR TITLE
User interface and map annotation improvements

### PR DESCRIPTION
**Pull request by nitinhemmady**
- Introduce a new alignment argument in the attributed string helper method
- Change map viewable area right edge insets
- Resolve ImageResource conflicts with Xcode 15 GeneratedAssetSymbols ImageResource class
- Resolve bottom linear player height on iPhone 15/Pro
- Resolve expanding annotation which lays out under other expended marker
- Gradually decrease PureLayout 3rd party dependencies
- Explicitly state SPM dependencies version number
- Fix audio player error view localization string
- Support new iPhone 15 models
- Update the MKMapView delegate method which returns the view associated with the specific annotation object
- Fix exhibitions cell subtitle height
- Increase build number
- Clean up code

### CODE REVIEW

File: aic/aic/Shared/Common.swift [hunk 1]
```diff
@@ -8,551 +8,602 @@ import CoreLocation
...
+            UIDevice().type == .iPhone13 ||
+            UIDevice().type == .iPhone14 ||
+            UIDevice().type == .iPhone14Plus ||
+            UIDevice().type == .iPhone14Pro ||
+            UIDevice().type == .iPhone14ProMax ||
+            UIDevice().type == .iPhone15 ||
+            UIDevice().type == .iPhone15Plus ||
+            UIDevice().type == .iPhone15Pro ||
+            UIDevice().type == .iPhone15ProMax
+        }()
```
Comment (nikhiltri) [inline aic/aic/Shared/Common.swift]: Does this need to be updated anytime Apple releases new phones? Is there a more streamlined way to determine this that doesn't require us to keep the list updated?

File: aic/aic/Shared/Common.swift [hunk 2]
```diff
@@ -8,551 +8,602 @@ import CoreLocation
...
+                                                 actionButtonTitle: "tour_leave_tour_leave_action".localized(using: "Base"),
+                                                 cancelButtonTitle: "tour_leave_tour_cancel_action".localized(using: "Base")
+        )
+
+        static let enableLocation = AICMessageModel(iconImage: #imageLiteral(resourceName: "messageMap"),
+                                                    title: "location_settings_header".localized(using: "LocationUI"),
+                                                    message: "location_settings_body".localized(using: "LocationUI"),
+                                                    actionButtonTitle: "global_accept_action".localized(using: "Base"),
+                                                    cancelButtonTitle: "global_cancel_action".localized(using: "Base")
+        )
```
Comment (nikhiltri) [inline aic/aic/Shared/Common.swift:159]: Are these messages translated if the device's language isn't English? I don't see these messages translated in this PR.

File: aic/aic/ViewControllers+Views/Sections/Map/AnnotationViews/MapObjectAnnotationView.swift [hunk 3]
```diff
@@ -9,11 +9,12 @@ import MapKit
 import Kingfisher
 
 protocol MapObjectAnnotationViewDelegate: AnyObject {
-	func mapObjectAnnotationViewPlayPressed(_ object: MapObjectAnnotationView)
+	func mapObjectAnnotationViewDidPressPlay(_ object: MapObjectAnnotationView)
```
Comment (nikhiltri) [inline aic/aic/ViewControllers+Views/Sections/Map/AnnotationViews/MapObjectAnnotationView.swift:12]: 👍🏾

File: aic/aic/ViewControllers+Views/Sections/Map/AnnotationViews/MapObjectAnnotationView.swift [hunk 4]
```diff
@@ -202,34 +141,33 @@ class MapObjectAnnotationView: MapAnnotationView {
 
 						self.backgroundView.backgroundColor = .white
 						self.transform = CGAffineTransform(scaleX: 1.0, y: 1.0)
-						self.bounds = self.backgroundView.frame.union(self.tailView.frame)
+						self.bounds = self.backgroundView.frame.union(self.tailView.frame) // ????
```
Comment (nikhiltri) [inline aic/aic/ViewControllers+Views/Sections/Map/AnnotationViews/MapObjectAnnotationView.swift]: Expand the comment to articulate what you're wondering.

**Review by nikhiltri — APPROVED** [review]
Looks good to me. ⭐ Approving with a few minor comments.

File: aic/aic/ViewControllers+Views/Sections/Map/AnnotationViews/MapObjectAnnotationView.swift (hunk 4 above)
Comment (nitinhemmady) [reply aic/aic/ViewControllers+Views/Sections/Map/AnnotationViews/MapObjectAnnotationView.swift]: Hi @nikhiltri,
There is not much to add here. It's leftover when I was investigating an annotation behaviour. 
Obsolete comment was removed. Thanks!

File: aic/aic/Shared/Common.swift (hunk 2 above)
Comment (nitinhemmady) [reply aic/aic/Shared/Common.swift:159]: Hi @nikhiltri,
It's a great point! Those messages are existing strings which are missing translations. Although, In this case I would suggest to find out if there is more missing translations and address it in a separate PR.

File: aic/aic/Shared/Common.swift (hunk 1 above)
Comment (nitinhemmady) [reply aic/aic/Shared/Common.swift]: Hi @nikhiltri,
The biometric identification check was improved.

**Pull request merged**